.rubocop.yml

```diff
@@ -1,29 +1,67 @@
-require: rubocop-rspec
+require: 
+  - rubocop-rspec
 
-Metrics/LineLength:
-  Max: 120
+AllCops:
+  Exclude:
+    - 'db/*'
+    - 'script/*'
+    - 'bin/*'
+    - 'node_modules/**/*'
 
 Metrics/MethodLength:
-  Max: 16
+  Max: 15
+  Exclude:
+    - 'db/**/*.rb'
+
+Metrics/BlockLength:
+  Exclude:
+    - 'Gemfile'
+    - 'spec/**/*.rb'
+    - config/initializers/*
+    - config/environments/*
+
+Metrics/LineLength:
+  Max: 145
+  Exclude:
+    - config/initializers/*
 
 Style/Documentation:
   Enabled: false
 
+Style/Semicolon:
+  Enabled: false  
+
+Style/MixinUsage:  
+  Enabled: false  
+
 Style/FrozenStringLiteralComment:
+  EnforcedStyle: never
```

Comment:
EnforcedStyle: always

or delete this two lines as it is a default option [https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment)